ImpSpec.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 2.4
 name: ImpSpec
-version: 0.2.0.0
+version: 0.2.1.0
```

If we are to follow ledger release policy, the this is incorrect. Version in cabal is bumped only when there are new changes after the last release
```suggestion
version: 0.2.0.0
```